Jointify/Views/Subviews/DefaultButton.swift

```diff
@@ -27,11 +27,13 @@ struct DefaultButton<ConformsToView: View>: View {
         Button(action: action, label: {
             self.label
                 .padding()
-                .background(Color.blue)
+                .background(Color.lightBlue)
                 .foregroundColor(.white)
```

Comment:
Can you also add the shade you guys proposed in the mockups, please? :-)

![image](https://user-images.githubusercontent.com/40395744/84954388-961a3f00-b0f5-11ea-8cef-9e6644510a3b.png)

Reply:
That is the shade I added with the custom color - I named it lightBlue ;)

Reply:
Do you have a screenshot for us? :)

Reply:
<img width="328" alt="grafik" src="https://user-images.githubusercontent.com/56828024/84955449-961b3e80-b0f7-11ea-981d-51bb5cf1c2e0.png">

Reply:
Sorry, I don't mean the colour but the downwards shade of the button on the screen.

Reply:
Ich meine den Schatten, nicht den Farbton 🙈 
![image](https://user-images.githubusercontent.com/40395744/84955897-64ef3e00-b0f8-11ea-9e51-ec3e8ca9e2e2.png)

Reply:
Ah sorry, shade in the sense of "Schatten" - I understood shade = "Farbton". Only 2 hours of sleep last to finish our paper isn't helping. Will tackle the complete UI tomorrow and Friday morning.

Reply:
Perfect, thanks a lot! 👍

Reply:
<img width="317" alt="grafik" src="https://user-images.githubusercontent.com/56828024/85053770-51e17a00-b19b-11ea-93d1-b70e0718d3da.png">
Added the shade, please approve merge request :)

Reply:
Looks great, thanks a lot!